plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/ServiceAccountUtil.java

```diff
@@ -41,11 +41,10 @@
    * @param destination path of a key file to be saved
    */
   public static void createServiceAccountKey(IGoogleApiFactory apiFactory,
-      Credential credential, String projectId, String serviceAccountId, Path destination)
+      Credential credential, String projectId, Path destination)
```

Comment:
If we remove the service account ID, we should rename this method to `createAppEngineDefaultServiceAccountKey` or document it properly, as it is no longer general. The Javadoc needs to be updated anyway.

Reply:
done